<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="loginPanel">
   <property name="styleSheet">
    <string notr="true">
    background-repeat: no-repeat;
    background-position: center;
    background-size: contain;</string>
   </property>
   <widget class="QLabel" name="backgroundLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  background-image: url(:/images/login.jpg);
    background-repeat: no-repeat;
    background-position: center;
    background-size: cover; /* یا contain */</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameEdit">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>171</y>
      <width>255</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: transparent;       
    border-style: solid;                    
    border-width: 2px;                   
    border-color: rgba(128, 0, 128, 1);  
    border-radius: 10px;               
    color: white; 
padding-left: 7px;
}

QLineEdit:focus {
    background-color: rgba(0, 0, 128, 0.5); 
    border-color: rgba(75, 0, 130, 0.8);    
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>215</y>
      <width>255</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: transparent;       
    border-style: solid;                    
    border-width: 2px;                   
    border-color: rgba(128, 0, 128, 1);  
    border-radius: 10px;               
    color: white; 
padding-left: 7px;
}

QLineEdit:focus {
    background-color: rgba(0, 0, 128, 0.5); 
    border-color: rgba(75, 0, 130, 0.8);    
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>262</y>
      <width>256</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: transparent;  /* پس‌زمینه شفاف */
   /* خط دور به رنگ بنفش */
    border-radius: 10px;            /* گوشه‌های گرد با شعاع 10 */
    color: white;   </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="goToSignupButton">
    <property name="geometry">
     <rect>
      <x>398</x>
      <y>309</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: transparent;  /* پس‌زمینه شفاف */
    border: 0.1px solid purple;       /* خط دور به رنگ بنفش */
    border-radius: 10px;            /* گوشه‌های گرد با شعاع 10 */
    color: white;   </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="forgotPasswordButton">
    <property name="geometry">
     <rect>
      <x>287</x>
      <y>354</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: transparent;  /* پس‌زمینه شفاف */

    border-radius: 10px;            /* گوشه‌های گرد با شعاع 10 */
    color: white;   </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
